kore/src/Kore/Step/Condition/Evaluator.hs

```diff
@@ -35,14 +25,11 @@ import           Kore.Step.Simplification.Data
                  ( PredicateSubstitutionSimplifier,
                  SimplificationProof (SimplificationProof), Simplifier,
                  StepPatternSimplifier (..) )
+import qualified Kore.Step.SMT.Evaluator as SmtEvaluator
```

Comment:
Why not have the import name match the end of the module name, `import ... as SMT.Evaluator`?